src/bucket.ts

```diff
@@ -1401,11 +1414,11 @@ class Bucket extends ServiceObject {
     callback?: CombineCallback
   ): Promise<CombineResponse> | void {
     if (!Array.isArray(sources) || sources.length === 0) {
-      throw new Error('You must provide at least one source file.');
+      throw new Error(BucketExceptionMessages.PROVIDE_SOURCE_FILE);
```

Comment:
Unrelated to the refactor: this looks like client side validation and it concerns me... Are these errors that we should be deferring to the server?